x/msgfees/types/codec.go

```diff
@@ -14,15 +13,6 @@ func RegisterInterfaces(registry cdctypes.InterfaceRegistry) {
 	messages := make([]proto.Message, len(allRequestMsgs))
 	copy(messages, allRequestMsgs)
 	registry.RegisterImplementations((*sdk.Msg)(nil), messages...)
-
-	registry.RegisterImplementations(
-		(*govtypesv1beta1.Content)(nil),
-		&AddMsgFeeProposal{},
-		&UpdateMsgFeeProposal{},
-		&RemoveMsgFeeProposal{},
-		&UpdateNhashPerUsdMilProposal{},
-		&UpdateConversionFeeDenomProposal{},
-	)
```

Comment:
There's a very good chance we need to keep this registration around forever.

My fear is that things will blow up when something (e.g. a gov proposals query) tries to read an old proposal with one of these in it.

Reply:
I will do a test for this.  The problem with registering them is that I need to keep the files around that implement the interfaces.  Not a big deal, but was hoping it wasn't needed.